src/pages/Contact.tsx

```diff
@@ -41,8 +46,27 @@ export default function Contact() {
         </div>
       </div>
       <div className="w-full h-full px-[3vw] py-[5vh] bg-[#f9f9f9] md:px-[6.5vw]">
-        <form className="w-full">
```

Comment:
For form you should be using form library like React-hook-form or formik

Reply:
Yes, my bad, I forgot to add it to the contact page as I was rushing. I used react hook form on other pages though, the login, sign up etc